### PR TITLE
add commands to list pending deals and force publish

### PR DESCRIPTION
**Pull request by dirkmc**
Fixes https://github.com/filecoin-project/lotus/issues/5491

### CODE REVIEW

File: api/apistruct/struct.go [hunk 1]
```diff
@@ -301,6 +301,8 @@ type StorageMinerStruct struct {
 		MarketDataTransferUpdates func(ctx context.Context) (<-chan api.DataTransferChannel, error)                                                                                                            `perm:"write"`
 		MarketRestartDataTransfer func(ctx context.Context, transferID datatransfer.TransferID, otherPeer peer.ID, isInitiator bool) error                                                                     `perm:"read"`
 		MarketCancelDataTransfer  func(ctx context.Context, transferID datatransfer.TransferID, otherPeer peer.ID, isInitiator bool) error                                                                     `perm:"read"`
+		MarketPendingDeals        func(ctx context.Context) (api.PendingDealInfo, error)                                                                                                                       `perm:"admin"`
```
Comment (arajasek) [inline api/apistruct/struct.go:304]: does this need `admin` perm?

**Review by arajasek — APPROVED** [review]
Seems wholly reasonable, just one non-blocking q.

File: api/apistruct/struct.go (hunk 1 above)
Comment (magik6k) [reply api/apistruct/struct.go:304]: Not really

**Comment by magik6k** [comment]
(tested on my miner; it works!)

**Pull request merged**